controlplane/migrations/0114_bright_carmella_unuscione.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "namespaces" ADD COLUMN "checks_timeframe_in_days" integer;
```

Comment:
Let's add a new table called "namespace_config" and you can add this column to the new table.

Reply:
We can even migrate the boolean values in the namespaces table to the new table

Reply:
My concern with this approach is that users that have already modified these options will have to do it again (even if just a simple boolean).

Reply:
We can migrate those values to the new table